features-json/webxr.json

```diff
@@ -350,9 +350,10 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Chrome 66-72 supported a very early version of the spec",
-    "2":"Part of an origin trial",
-    "3":"Part of the spec can be enabled with the [webxr](chrome://flags/#webxr) and [webxr-hit-test](chrome://flags/#webxr-hit-test) flags."
+    "1":"Chrome 66-72 supported a very early version of the spec.",
```

Comment:
These versions are not usable without flags, which are only intended for developers who want to try the thing out. Since the API will soon be enabled by default, the early version isn't anything anyone should be using.

Reply:
Again, this does not help developers build for the future. I recommend removing it.